bia_export/config.py

```diff
@@ -12,8 +12,13 @@ class Settings(BaseSettings):
     bioformats2raw_bin: str = ""
     config_fpath: Path = Path("dome.yaml")
 
+    bia_api_basepath: str = 'https://bia-cron-1.ebi.ac.uk:8080'
```

Comment:
I know this is outside the lines you have changed, but endpoint_url (line 8) and some of the other value names look worse now they are close to this one that has been named clearly.

Reply:
Is it the lack of namespacing? The names in lines 8-12 are a throwback to code written in the early days of the bia-integrator. We can discuss standardising naming conventions at the next code review.